mlir/include/mlir/Dialect/LLVMIR/ROCDLOps.td

```diff
@@ -192,6 +192,18 @@ def ROCDL_BarrierOp : ROCDL_Op<"barrier"> {
   let assemblyFormat = "attr-dict";
 }
 
+def ROCDL_SetPrioOp : ROCDL_IntrOp<"s.setprio", [], [], [], 0>,
+  Arguments<(ins I16:$priority)> {
```

Comment:
... Yeah, same observation here as below. LLVM says this has to be a constant, might as well make at an attribute on this side.